BE/OwnershipAndControl/Executives.rdf

```diff
@@ -382,7 +416,7 @@
 		<rdfs:subClassOf rdf:resource="&fibo-be-oac-exec;BoardMember"/>
 		<rdfs:subClassOf rdf:resource="&fibo-be-oac-exec;Executive"/>
 		<rdfs:label>executive board member</rdfs:label>
-		<skos:definition>executive that is also a member of the board of directors of the organization they manage</skos:definition>
+		<skos:definition>executive that is also a director of the organization they manage</skos:definition>
```

Comment:
(trivial) I think better reversed. Also don't confuse "board member" with "director" (see other comment). IOW "board member that is also an executive..."
Note though I just discovered that this term is also used as a specific job title e.g. instead of a CEO a non-profit might have an Executive Director see https://smallbusiness.chron.com/difference-between-executive-director-chief-executive-officer-21800.html

Reply:
I've reworded it along the lines that you were looking for.